Tests/SentryTests/Integrations/Performance/IO/SentryFileIOTrackingIntegrationTests.swift

```diff
@@ -1,11 +1,8 @@
 import Foundation
 import Sentry
+import SentryTestUtils
 import XCTest
- 
-// This test is also executed under iOS-SwiftUITests, because
-// GitHub Actions doesn't have simulators for iOS 11 and 10.
-// That's why we need to keep it generic, without access
-// to any private part of the SDK.
```

Comment:
This didn't make sense to me. If we have no iOS 10/11 simulator for unit tests, how would we have one for UI tests? I removed this file from the iOS-SwiftUITest target.

Reply:
UI Tests runs under SouceLabs that contains iOS 11 and 10, but im fine by removing this from ui tests.

Reply:
ahhh that’s right, totally forgot about saucelabs 😂